device-connectors/src/testflinger_device_connectors/devices/oem_autoinstall/oem_autoinstall.py

```diff
@@ -185,3 +215,22 @@ def hardreset(self):
                 subprocess.check_call(cmd.split(), timeout=120)
             except subprocess.SubprocessError as exc:
                 raise RecoveryError("Error running reboot script!") from exc
+
+    def check_device_booted(self):
+        """Check to see if the device is booted and reachable with ssh"""
+        logger.info("Checking to see if the device is available.")
+        started = time.time()
+        # Wait for provisioning to complete - can take a very long time
+        while time.time() - started < 3600:
```

Comment:
I suggest that we should increase the waiting time to 90 mins:
- image size becomes larger.
- network stability

Reply:
OK, increased this to 90 min.